packages/manager/apps/key-management-service/src/pages/serviceKey/createKey.page.tsx

```diff
@@ -0,0 +1,413 @@
+                  )}
+                </CommonTitle>
+                <Description>
+                  {t(
+                    'key_management_service_service-keys_create_crypto_field_usage_subtitle',
+                  )}
+                </Description>
+                <OsdsRadioGroup>
+                  {key?.operations.map((operation) => {
+                    return (
+                      <OsdsRadio
+                        value={operation.value.join(' / ')}
+                        key={operation.value.join(' / ')}
+                        checked={
+                          operation.value.join(' / ') ===
+                          keyOperation?.join(' / ')
+                        }
+                        className="mb-6"
+                      >
+                        <OsdsRadioButton
+                          disabled={
+                            key.type === OkmsKeyTypes.EC ||
+                            key.type === OkmsKeyTypes.RSA
+                          }
+                          size={ODS_RADIO_BUTTON_SIZE.sm}
+                          color={ODS_THEME_COLOR_INTENT.primary}
+                          onClick={() => {
+                            setKeyOperation(operation.value);
+                          }}
+                        >
+                          <span slot="end">
+                            <ServiceKeyOperations
+                              operations={operation.value}
+                              text
+                            />
+                            <Description>
+                              {t(
+                                `key_management_service_service-keys_create_crypto_field_usage_description_${operation.value.join(
+                                  '_',
+                                )}`,
+                              )}
+                            </Description>
+                          </span>
+                        </OsdsRadioButton>
+                      </OsdsRadio>
+                    );
+                  })}
+                </OsdsRadioGroup>
+              </div>
+            </div>
+            <OsdsDivider />
+            <OsdsButton
+              size={ODS_BUTTON_SIZE.md}
+              inline
+              variant={ODS_BUTTON_VARIANT.stroked}
+              color={ODS_THEME_COLOR_INTENT.primary}
+              className={'xs:mt-2 sm:mt-0 w-fit h-fit'}
+              onClick={() => {
+                navigate(`/${okmsId}${ROUTES_URLS.keys}`);
+              }}
+            >
+              {t('key_management_service_service-keys_create_cta_cancel')}
+            </OsdsButton>
+            <OsdsButton
+              size={ODS_BUTTON_SIZE.md}
+              inline
+              color={ODS_THEME_COLOR_INTENT.primary}
+              className={'xs:mt-2 sm:mt-0 sm:ml-4 w-fit h-fit'}
+              onClick={submitCreateKey}
+              disabled={!!serviceKeyNameError || isPending || undefined}
```

Comment:
Is `false` not enough ?
```suggestion
              disabled={!!serviceKeyNameError || isPending }
```